src/transformState.js

```diff
@@ -62,7 +62,29 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const transform of transforms) {
+    const { operation, properties } = transform;
```

Comment:
```suggestion
  for (const { operation, properties }  of transforms) {
```
Можна робити деструктуризацію прямо тут